packages/devkit/index.ts

```diff
@@ -319,3 +319,8 @@ export { Hash, Hasher } from 'nx/src/hasher/hasher';
  * @category Utils
  */
 export { cacheDir } from 'nx/src/utils/cache-directory';
+
+/**
+ * @category Utils
+ */
+export { createIgnoreFromFS, createIgnoreFromTree } from 'nx/src/utils/ignore';
```

Comment:
I don't think we need to re-export these, since it isn't likely relevant for community plugins. If someone requests it, we can re-evaluate but for now can you remove them?

Reply:
Hmmm, now that you mention this I think I can use it in nx-dotnet too.